.travis.yml

```diff
@@ -3,26 +3,41 @@ language: python
 python:
     - 2.7
```

Comment:
Oh - no need to work on this for this PR, but can you please issue us with upgrading to python 3, please?